qiita_db/util.py

```diff
@@ -1486,17 +1488,16 @@ def generate_study_list(user, visibility):
 
                 # cleaning aids_with_deprecation
                 info['artifact_biom_ids'] = []
-                info['preparation_data_types'] = []
                 if info['aids_with_deprecation'] is not None:
                     for x in info['aids_with_deprecation']:
-                        # f1-2 are the default names given by pgsql
+                        # f1-3 are the default names given by pgsql
```

Comment:
Just to confirm, f1-3 I assume means (f1, f2, f3). You've increased the range by 1, but the conditional below is still testing only for not x['f2']. This is still okay, right?

Reply:
Yeah, good catch. Originally we had 3 but the comment only mentioned 2, then changed the code to only use 2 and forgot to bring the comment back ... changing ...